### PR TITLE
BQ DWH the very first version

### PR DESCRIPTION
**Pull request by meken**

### CODE REVIEW

**Comment by gfilicetti** [comment]
For walking through.... [staged student guide is here](https://meken.github.io/cloud-hackathons/)

**Review by gfilicetti — APPROVED** [review]
Beautiful work, once again.

Just a few little quibbles. I approved so you can merge without waiting for me on Monday morning.

Time for the weekend to start.

File: hacks/bq-dwh/README.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+
+- Challenge 1: Loading the source data
+- Challenge 2: Staging tables
+- Challenge 3: Dataform for automation
+- Challenge 4: Dimensional modeling
+- Challenge 5: Business Intelligence
+- Challenge 6: Access control
+- Challenge 7: Notebooks for data scientists
+- Challenge 8: Cloud Composer for orchestration
+- Challenge 9: Monitoring the workflow
```
[the remark this answers is not in the document]
Comment (meken) [reply hacks/bq-dwh/README.md:35]: :shrug: these were late additions, I ran out of witty ideas :smile:

File: hacks/bq-dwh/README.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+
+## Challenge 8: Cloud Composer for orchestration
+
+### Introduction
+
+Running the Dataform pipelines manually works, but it's not very practical. We'd rather automate this process and run it periodically. Although Dataform provides a lot of functionality to automate and schedule running the pipelines, we're going to consider a bit more flexible orchestrator  that can also run additional steps that might not be part of the Dataform pipelines, such as pulling data from source systems, running the inferencing with the model we've created in the last challenge etc.
+
+This challenge is all about Cloud Composer, which is basically a managed and serverless version of the well-known [Apache Airflow](https://airflow.apache.org/) framework, to schedule and run our complete pipeline.
+
+> **Note** There's a myriad of different orchestration services on Google Cloud, see the [documentation](https://cloud.google.com/bigquery/docs/orchestrate-workloads) for more information and guidance on which one to pick for *your* specific needs.
```
[the remark this answers is not in the document]
Comment (meken) [reply hacks/bq-dwh/README.md:291]: Yes, the link gives an overview of all the available orchestrators including Workflows. Even though Airflow is a bit of an overkill here, it's the de-factor standard for Data Engineering hence the choice.

**Review by gfilicetti — APPROVED** [review]
Brilliance

**Pull request merged**